fullscreen.bs

```diff
@@ -61,8 +61,16 @@ stated otherwise it is unset.
 <p>All <a for=/>documents</a> have an associated <dfn>list of pending fullscreen events</dfn>, which
 is an <a>ordered set</a> of (<a>string</a>, <a>element</a>) <a>tuples</a>. It is initially empty.
 
-<p>To <dfn>fullscreen an <var>element</var></dfn>, set <var>element</var>'s <a>fullscreen flag</a>
-and <a for="top layer">add</a> it to its <a>node document</a>'s <a>top layer</a>.
+<p>To <dfn>fullscreen an <var>element</var></dfn>:
+
+<ol>
+ <li><p>Run {hide all popovers}.
```

Comment:
This ends up not linking to https://html.spec.whatwg.org/#hide-all-popovers, let's try:

```suggestion
 <li><p>Run <a>hide all popovers</a>.
```